Assignment2/xelabravo94.rdf

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:ns0="http://www.ontogrid.net/StickyNote#">
+
+  <rdf:Description rdf:about="http://www.ontogrid.net/StickyNote#Class01">
+    <ns0:includes>
+      <rdf:Description rdf:about="http://www.ontogrid.net/StickyNote#Computer101">
+        <ns0:hasOwner>
+          <rdf:Description rdf:about="http://www.ontogrid.net/StickyNote#User10A">
+            <ns0:hasName rdf:resource="http://www.ontogrid.net/StickyNote#Pedro"/>
+          </rdf:Description>
+        </ns0:hasOwner>
+
+      </rdf:Description>
+    </ns0:includes>
+
+    <ns0:includes>
+      <rdf:Description rdf:about="http://www.ontogrid.net/StickyNote#Sensor029">
+        <ns0:hasMeasurement>
+          <rdf:Description rdf:about="http://www.ontogrid.net/StickyNote#Measurement8401">
+            <ns0:hasTemperature rdf:resource="http://www.ontogrid.net/StickyNote#29"/>
+            <ns0:atTime rdf:resource="http://www.ontogrid.net/StickyNote#2010-06-12T 12:00:12"/>

```

Comment:
Be careful with spaces in the data. It should not be a resource, but a literal. Same with the another value.